GPU/GPUTracking/DataCompression/GPUTPCCompressionKernels.cxx

```diff
@@ -87,7 +87,7 @@ GPUdii() void GPUTPCCompressionKernels::Thread<GPUTPCCompressionKernels::step0at
       if (nClustersStored == 1) {
         unsigned char qpt = fabs(trk.GetParam().GetQPt()) < 20.f ? (trk.GetParam().GetQPt() * (127.f / 20.f) + 127.5f) : (trk.GetParam().GetQPt() > 0 ? 254 : 0);
         zOffset = z;
-        track.Init(x, y, z - zOffset, param.SliceParam[hit.slice].Alpha, qpt, param);
```

Comment:
Why do you remove the zOffset here?
This is actually not ideal and will be replaced by a better zOffset estimation, but we need to keep a zOffset for long time frames, and using z works to first order.
Of course you can force it to 0, but that is insufficient in the long run when we plan to put in a more reasonable estimate.

Reply:
I did not know that zOffset will be replaced later with a better estimate. I looked at the current implementation and thought that if zOffset is set to z in any case I can replace z - zOffset with zero.. But I will change this back now